test_runner/src/test/kotlin/ftl/util/UtilsTest.kt

```diff
@@ -197,8 +197,7 @@ class UtilsTest {
         // will
         exit.expectSystemExitWithStatus(3)
         exit.checkAssertionAfterwards {
-            assertTrue(output.log.contains("More details are available at [${testMatrix1.webLink}]"))
-            assertOutputIsEqualReferenceTable(output.log, testMatrix1.matrixId)
+            assertTrue(output.log.contains("Matrix is ${testMatrix1.state}"))
```

Comment:
There will be no table output if flank throw `FTLError` before `MatrixResultsReport`?